AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -217,21 +219,22 @@ func addProtocolTests() {
         .init(name: "S3TestSDK", sourcePath: "\(baseDir)/s3"),
         .init(name: "rest_json_extras", sourcePath: "\(baseDirLocal)/rest_json_extras"),
         .init(name: "AwsQueryExtras", sourcePath: "\(baseDirLocal)/AwsQueryExtras"),
+        .init(name: "EventStream", sourcePath: "\(baseDirLocal)/EventStream", buildOnly: true),
+        .init(name: "RPCEventStream", sourcePath: "\(baseDirLocal)/RPCEventStream", buildOnly: true),
```

Comment:
Added the ability to add these two Smithy service definitions to local protocol tests.

These two Smithy services don't have any protocol tests, but at least this will build them & verify that the code compiles.

Had to add a little logic here to allow adding protocol test targets to the manifest that don't have actual test bundles.